content/boundary/v0.21.x/content/docs/credentials/configure-credential-injection.mdx

```diff
@@ -32,8 +30,18 @@ Credential injection is supported for:
 
 - You must have a static credential saved in your static credential store or Vault credential store. The credential must correspond to the target to which you want to authenticate.
 
-- **For RDP targets**: Network Level Authentication (NLA) is supported. Kerberos and NTLMv2 authentication methods are supported for domain-joined workers. NTLMv2 is supported for non-domain-joined workers.
-- **For SSH targets**: Keyboard-interactive authentication is not supported. When you use username-password credentials, ensure that your SSH server is configured to allow password authentication.
+- Specific target types have the following unique requirements:
+  - **For RDP targets**: Network Level Authentication (NLA) is supported. Kerberos and NTLMv2 authentication methods are supported for domain-joined workers. NTLMv2 is supported for non-domain-joined workers.
+
+    RDP credential injection also has the following specific network requirements:
+
+    - UDP transport must be disabled.
+    - Server redirection is not supported.
+    - The maximum supported TLS version is 1.2. TLS 1.3 is incompatible with Windows Server 2025.
```

Comment:
iirc there was a golang issue somewhere about why specifically TLS 1.3 didn't work for windows server 2025. @irenarindos any chance you remember what that was?